components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_connection_api.cc

```diff
@@ -58,12 +58,6 @@ void BraveVPNWireguardConnectionAPI::CheckConnection() {
   UpdateAndNotifyConnectionStateChange(state);
 }
 
-void BraveVPNWireguardConnectionAPI::RequestNewProfileCredentials() {
-  brave_vpn::wireguard::WireguardGenerateKeypair(base::BindOnce(
```

Comment:
maybe we could delete `brave_vpn::wireguard::WireguardGenerateKeypair()` from wireguard_utils_win.h?

Reply:
I want to do this as separate PR later when we test everything well